### PR TITLE
fix aggregate bonded transaction error "Failure_Core_Future_Deadline"

### LINKED ISSUE #1490: Aggegate bonded - Failure_Core_Future_Deadline
Steps:

1. Change system clock +1h
2. Try to send aggregate bonded transaction (for example multisig)

Result: Announcing hash lock failes with error "Failure_Core_Future_Deadline":

image

Questions:

1) Why Hash lock transaction deadline is set to 6 hours? Hash lock is confirmed immediately. Why it has 6h instead of 2h like a normal transaction?

2) When the user operating system clock is ahead 1minute (or node clock 1minute behind) the user gets an error as above. I assume here that the max deadline for the hash lock is 6h but it seems correct assumption.

3) Wouldn't better to use node time (/node/time) instead of local user time?

### PR DESCRIPTION
**Pull request by bassemmagdy**
- fix #1490 
- upgrading to newer alpha-sdk

### CODE REVIEW

File: src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts [hunk 1]
```diff
@@ -466,9 +471,11 @@ export class ModalTransactionConfirmationTs extends Vue {
 
     private async ledgerAccountMultisigTransactionOnSigner(values) {
         const { ledgerService, currentPath, isOptinLedgerWallet, ledgerAccount, multisigAccount, stageTransactions, maxFee } = values;
+        await this.$store.dispatch('network/SET_TRANSACTION_DEADLINE', 42, { root: true });
```
Comment (rg911) [inline src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts:474]: why using 42 here? Max aggregate transaction deadline can be 48 hours

Comment (bassemmagdy) [reply src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts:474]: we've been discussing this, ended up decreasing them(LockHash and Aggregate) to 4h and 42h for a more safer option as the network might reject when the local time is slightly unsynchronized.

Comment (cryptoBeliever) [reply src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts:474]: @bassemmagdy
1) Why we used slightly unsynchronized delta for hashlock to 2h (6-4) and for Aggregate 6h (48-42). I don't think it has any sense.
2) Aren't we switched to node time instead local time? There should be no "slightly unsynchronized" then. So we should use 6 and 48 in my opinion. Do you see reason to not use it?

File: src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts [hunk 2]
```diff
@@ -480,9 +487,11 @@ export class ModalTransactionConfirmationTs extends Vue {
             .then((signedAggregateTransaction) => {
                 return signedAggregateTransaction;
             });
+        await this.$store.dispatch('network/SET_TRANSACTION_DEADLINE', 4, { root: true });
```
[the remark this answers is not in the document]
Comment (cryptoBeliever) [reply src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts:490]: Same as in https://github.com/nemgrouplimited/symbol-desktop-wallet/pull/1516#discussion_r650365632

File: src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts (hunk 1 above)
Comment (bassemmagdy) [reply src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts]: got your point and it makes sense updated them set deadlines to max again, thanks!

**Comment by bassemmagdy** [comment]
updated here #1552 .

**Pull request closed without merging**